resources/nss.rb

```diff
@@ -12,6 +12,8 @@
     binary_source = "libnss_cache.so.2.0_1404"
   elsif node['platform'] == 'ubuntu' && node['platform_version'] == '16.04'
     binary_source = "libnss_cache.so.2.0_1604"
+  elsif node['platform'] == 'ubuntu' && node['platform_version'] == '20.04'
+    binary_source = "libnss_cache.so.2.0_2004"
```

Comment:
can we please add a small note on the steps to get these binaries/build them so that they can be looked up easily

Reply:
would we also require to add a binary for pam_gate similar to this one https://github.com/gate-sso/chef_gate/blob/master/files/default/pam_gate.so_1604 for 20.04?

Reply:
not required since gate_wrapper does not use the `pam_gate`.

Reply:
added a BINARYGENERATION guide.